src/pages/ProvideLiquidity/LiquidityCard.tsx

```diff
@@ -72,24 +72,24 @@ export default function LiquidityCard({
     setActionType(type)
   }
 
-  if (!token || !rfpToken) {
+  if (!token0 || !token1) {
     return null
   }
 
   return (
     <PoolCard
-      token0={token}
-      token1={rfpToken}
+      token0={token0}
```

Comment:
Is support for displaying the APY for these metafarms going to be added in a separate PR? (I'm happy to work on that also!)

Reply:
yes! i didn't have time to do the APY calc for this but if you'd like to take it then feel free!